### ISSUE TITLE
Getting loss = nan after some iterations

### ISSUE THREAD

**Issue by alchemician** [Zielon/INSTA#17]
Which parameters can I change to avoid this? Any suggested parameter values?

16:35:16 INFO     iteration=858 loss=0.004551
16:35:16 INFO     iteration=859 loss=0.004551
16:35:16 INFO     iteration=860 loss=0.004551
16:35:16 INFO     iteration=861 loss=0.004551
16:35:16 INFO     iteration=862 loss=0.004551
16:35:16 INFO     iteration=863 loss=0.004551
16:35:16 INFO     iteration=864 loss=0.004551
16:35:16 INFO     iteration=865 loss=nan
16:35:16 INFO     iteration=866 loss=nan
16:35:16 INFO     iteration=867 loss=nan
16:35:16 INFO     iteration=868 loss=nan
16:35:16 INFO     iteration=869 loss=nan
16:35:16 INFO     iteration=870 loss=nan
16:35:16 INFO     iteration=871 loss=nan
16:35:16 INFO     iteration=872 loss=nan

**Comment by alchemician** [comment]
# packages in environment at /home/ubuntu/anaconda3/envs/rta:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             4.5                       1_gnu  
addict                    2.4.0                    pypi_0    pypi
aiohttp                   3.8.3                    pypi_0    pypi
aiosignal                 1.2.0                    pypi_0    pypi
anyio                     3.6.1                    pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
argon2-cffi               21.3.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
asttokens                 2.0.5                    pypi_0    pypi
async-timeout             4.0.2                    pypi_0    pypi
attrs                     22.1.0                   pypi_0    pypi
av                        8.0.3                    pypi_0    pypi
babel                     2.10.3                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
beautifulsoup4            4.11.1                   pypi_0    pypi
blas                      1.0                         mkl  
bleach                    5.0.1                    pypi_0    pypi
brotli                    1.0.9                he6710b0_2  
brotlipy                  0.7.0           py39h27cfd23_1003  
bzip2                     1.0.8                h7b6447c_0  
c-ares                    1.18.1               h7f8727e_0    anaconda
ca-certificates           2023.01.10           h06a4308_0    anaconda
certifi                   2022.12.7        py39h06a4308_0    anaconda
cffi                      1.15.1                   pypi_0    pypi
charset-normalizer        2.1.0                    pypi_0    pypi
cmake                     3.22.1               h1fce559_0    anaconda
commonmark                0.9.1                    pypi_0    pypi
cryptography              39.0.1           py39h9ce1e76_0  
cuda-cudart               11.7.99                       0    nvidia
cuda-cupti                11.7.101                      0    nvidia
cuda-libraries            11.7.1                        0    nvidia
cuda-nvrtc                11.7.99                       0    nvidia
cuda-nvtx                 11.7.91                       0    nvidia
cuda-runtime              11.7.1                        0    nvidia
cycler                    0.11.0             pyhd3eb1b0_0  
dbus                      1.13.18              hb2f20db_0  
debugpy                   1.6.2                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
deprecation               2.1.0                    pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
executing                 0.9.1                    pypi_0    pypi
expat                     2.4.4                h295c915_0  
fastjsonschema            2.16.1                   pypi_0    pypi
ffmpeg                    4.3                  hf484d3e_0    pytorch
filelock                  3.9.0            py39h06a4308_0  
flit-core                 3.6.0              pyhd3eb1b0_0  
fontconfig                2.13.1               h6c09931_0  
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.11.0               h70c0345_0  
frozenlist                1.3.1                    pypi_0    pypi
giflib                    5.2.1                h7b6447c_0  
glib                      2.69.1               h4ff587b_1  
gmp                       6.2.1                h295c915_3  
gmpy2                     2.1.2            py39heeb90bb_0  
gnutls                    3.6.15               he1e5248_0  
gst-plugins-base          1.14.0               h8213a91_2  
gstreamer                 1.14.0               h28cd5cc_2  
icu                       58.2                 he6710b0_3  
idna                      3.3                      pypi_0    pypi
imageio                   2.19.3                   pypi_0    pypi
importlib-metadata        4.12.0                   pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561  
ipykernel                 6.15.1                   pypi_0    pypi
ipython                   8.4.0                    pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                7.7.1                    pypi_0    pypi
jedi                      0.18.1                   pypi_0    pypi
jinja2                    3.1.2            py39h06a4308_0  
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9e                   h7f8727e_0  
json5                     0.9.8                    pypi_0    pypi
jsonschema                4.9.0                    pypi_0    pypi
jupyter-client            7.3.4                    pypi_0    pypi
jupyter-core              4.11.1                   pypi_0    pypi
jupyter-packaging         0.12.2                   pypi_0    pypi
jupyter-server            1.18.1                   pypi_0    pypi
jupyterlab                3.4.4                    pypi_0    pypi
jupyterlab-pygments       0.2.2                    pypi_0    pypi
jupyterlab-server         2.15.0                   pypi_0    pypi
jupyterlab-widgets        1.1.1                    pypi_0    pypi
kiwisolver                1.4.2            py39h295c915_0  
krb5                      1.19.2               hac12032_0    anaconda
lame                      3.100                h7b6447c_0  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.35.1               h7274673_9  
libcublas                 11.10.3.66                    0    nvidia
libcufft                  10.7.2.124           h4fbf590_0    nvidia
libcufile                 1.6.0.25                      0    nvidia
libcurand                 10.3.2.56                     0    nvidia
libcurl                   7.82.0               h0b77cf5_0    anaconda
libcusolver               11.4.0.1                      0    nvidia
libcusparse               11.7.4.91                     0    nvidia
libedit                   3.1.20210910         h7f8727e_0    anaconda
libev                     4.33                 h7f8727e_1    anaconda
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.3.0               h5101ec6_17  
libgomp                   9.3.0               h5101ec6_17  
libiconv                  1.16                 h7f8727e_2  
libidn2                   2.3.2                h7f8727e_0  
libnghttp2                1.46.0               hce63b2e_0    anaconda
libnpp                    11.7.4.75                     0    nvidia
libnvjpeg                 11.8.0.2                      0    nvidia
libpng                    1.6.37               hbc83047_0  
libssh2                   1.10.0               h8f2d780_0    anaconda
libstdcxx-ng              9.3.0               hd4cf53a_17  
libtasn1                  4.16.0               h27cfd23_0  
libtiff                   4.2.0                h2818925_1  
libunistring              0.9.10               h27cfd23_0  
libuuid                   1.0.3                h7f8727e_2  
libuv                     1.40.0               h7b6447c_0    anaconda
libwebp                   1.2.2                h55f646e_0  
libwebp-base              1.2.2                h7f8727e_0  
libxcb                    1.15                 h7f8727e_0  
libxml2                   2.9.14               h74e7548_0  
lpips                     0.1.4                    pypi_0    pypi
lz4-c                     1.9.3                h295c915_1  
markupsafe                2.1.1            py39h7f8727e_0  
matplotlib                3.5.1            py39h06a4308_1  
matplotlib-base           3.5.1            py39ha18d171_1  
matplotlib-inline         0.1.3                    pypi_0    pypi
meshio                    5.3.4                    pypi_0    pypi
mistune                   0.8.4                    pypi_0    pypi
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0            py39h7f8727e_0  
mkl_fft                   1.3.1            py39hd3c417c_0  
mkl_random                1.2.2            py39h51133e4_0  
mpc                       1.1.0                h10f8cd9_1  
mpfr                      4.0.2                hb69a4c5_1  
mpmath                    1.2.1            py39h06a4308_0  
multidict                 6.0.2                    pypi_0    pypi
munkres                   1.1.4                      py_0  
nbclassic                 0.4.3                    pypi_0    pypi
nbclient                  0.6.6                    pypi_0    pypi
nbconvert                 6.5.0                    pypi_0    pypi
nbformat                  5.4.0                    pypi_0    pypi
ncurses                   6.3                  h7f8727e_2  
nest-asyncio              1.5.5                    pypi_0    pypi
nettle                    3.7.3                hbbd107a_1  
networkx                  2.8.5                    pypi_0    pypi
notebook                  6.4.12                   pypi_0    pypi
notebook-shim             0.1.0                    pypi_0    pypi
numpy                     1.22.3           py39he7a7128_0  
numpy-base                1.22.3           py39hf524024_0  
open3d                    0.15.2                   pypi_0    pypi
opencv-python             4.6.0.66                 pypi_0    pypi
openh264                  2.1.1                h4ff587b_0  
openssl                   1.1.1s               h7f8727e_0    anaconda
packaging                 21.3               pyhd3eb1b0_0  
pandas                    1.4.3                    pypi_0    pypi
pandocfilters             1.5.0                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pcre                      8.45                 h295c915_0  
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    9.1.0                    pypi_0    pypi
pims                      0.5                      pypi_0    pypi
pip                       21.2.4           py39h06a4308_0  
prometheus-client         0.14.1                   pypi_0    pypi
prompt-toolkit            3.0.30                   pypi_0    pypi
psutil                    5.9.1                    pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0  
pygments                  2.12.0                   pypi_0    pypi
pyopenssl                 23.0.0           py39h06a4308_0  
pyparsing                 3.0.4              pyhd3eb1b0_0  
pyqt                      5.9.2            py39h2531618_6  
pyquaternion              0.9.9                    pypi_0    pypi
pyrsistent                0.18.1                   pypi_0    pypi
pysocks                   1.7.1            py39h06a4308_0  
python                    3.9.12               h12debd9_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
pytorch-cuda              11.7                 h778d358_3    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2022.1                   pypi_0    pypi
pyvista                   0.36.1                   pypi_0    pypi
pywavelets                1.3.0                    pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     23.2.0                   pypi_0    pypi
qt                        5.9.7                h5867ecd_1  
readline                  8.1.2                h7f8727e_1  
requests                  2.28.1           py39h06a4308_1  
rhash                     1.4.1                h3c74f83_1    anaconda
rich                      12.6.0                   pypi_0    pypi
scikit-image              0.19.3                   pypi_0    pypi
scikit-learn              1.1.1                    pypi_0    pypi
scipy                     1.9.0                    pypi_0    pypi
scooby                    0.6.0                    pypi_0    pypi
send2trash                1.8.0                    pypi_0    pypi
setuptools                61.2.0           py39h06a4308_0  
sip                       4.19.13          py39h295c915_0  
six                       1.16.0             pyhd3eb1b0_1  
slicerator                1.1.0                    pypi_0    pypi
sniffio                   1.2.0                    pypi_0    pypi
soupsieve                 2.3.2.post1              pypi_0    pypi
sqlite                    3.38.2               hc218d9a_0  
stack-data                0.3.0                    pypi_0    pypi
sympy                     1.11.1           py39h06a4308_0  
terminado                 0.15.0                   pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2022.7.28                pypi_0    pypi
tinycss2                  1.1.1                    pypi_0    pypi
tk                        8.6.11               h1ccaba5_0  
tomlkit                   0.11.1                   pypi_0    pypi
torch                     1.12.1                   pypi_0    pypi
torchaudio                2.0.0                py39_cu117    pytorch
torchtriton               2.0.0                      py39    pytorch
torchvision               0.13.1                   pypi_0    pypi
tornado                   6.1              py39h27cfd23_0  
tqdm                      4.64.0                   pypi_0    pypi
traitlets                 5.3.0                    pypi_0    pypi
trimesh                   3.11.2                   pypi_0    pypi
typing-extensions         4.3.0                    pypi_0    pypi
tzdata                    2022a                hda174b7_0  
urllib3                   1.26.11                  pypi_0    pypi
vtk                       9.2.2                    pypi_0    pypi
wcwidth                   0.2.5                    pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.3.3                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0  
widgetsnbextension        3.6.1                    pypi_0    pypi
wslink                    1.8.2                    pypi_0    pypi
xz                        5.2.5                h7b6447c_0  
yarl                      1.8.1                    pypi_0    pypi
zipp                      3.8.1                    pypi_0    pypi
zlib                      1.2.12               h7f8727e_2  
zstd                      1.5.2                ha4553b6_0

**Comment by Zielon** [comment]
It is difficult to say w/o knowing what kind of sequence you are training. Make sure that the tracking worked correctly and eventually play a bit with hyperparameters for the instant-ngp.
